CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# MIOpen Contributing Guide
+To ensure the quality of the MIOpen code base, the MIOpen team has 
+established a code review process to inform developers of the steps 
+that are required to shepherd a change-set into the repository.
+
+## Creating a Pull Request
+No changes are allowed to be directly committed to the develop 
+branch of the MIOpen repository. All authors are required to 
+develop their change sets on a separate branch and then create 
+a pull request (PR) to merge their changes into the develop branch.
+
+Once a PR has been created, a developer must choose two reviewers 
+to review the changes made. The first reviewer should be a 
+technical expert in the portion of the library that the changes 
+are being made in. You can find a list of these experts in 
+[MIOpen Issue #789](https://github.com/ROCmSoftwarePlatform/MIOpen/issues/789)
+. The second reviewer should be a peer reviewer. This reviewer 
+can be any other MIOpen developer.
+
+## Responsibility of Author
```

Comment:
Do we have code style guidelines?

Reply:
@atamazov, do we have a style available? I did find a page on our wiki related to [using Clang Format](https://github.com/ROCmSoftwarePlatform/MIOpen/wiki/How-to-format-code)

Reply:
No, we do not have a dedicated coding style doc. Tidy and formatting checks on CI provide the basis for the programming style automatically. I would also recommend reading https://isocpp.org/wiki/faq/coding-standards and https://google.github.io/styleguide/cppguide.html for every MIOpen developer.